chains/superpositiontestnet/metadata.yml

```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://testnet-explorer.superposition.so/api
+    family: blockscout
+    name: CatScan
+    url: https://testnet-explorer.superposition.so/
```

Comment:
Definitely not a big deal but if you don't mind, please remove the trailing slash `/`